help/en/html/doc/Technical/JVMCapabilities.shtml

```diff
@@ -211,14 +211,18 @@
 
       </table>
       (* indicates that a compatibility library is used in the early version)
+
       <h2>JRE availability</h2>
 
       <p>This section lists the most recent Java Runtime version recommended by Oracle for various
       operating system versions. Please note that not all users of that operating system will have
       updated to this Java version; many will be using an older one.</p>
 
+      <p>Java installers are now available from mlutiple sources.  Our
+        <a href="/java/index.shtml">Java installer page</a> lists a number of them.
+
       <p>JMRI development requires Java 8u101 or later through January 2022. 8u101 contains a certificate change that's
-      needed to link to some remote Javadocs. After that, Java 11 will be required.</p>
+      needed to link to some remote Javadocs. After January 2022, Java 11 will be required.</p>
```

Comment:
No big deal and I'm happy with it as is.
But it's December now and I suppose we will merge j11master into master before the end of December. So I guess that it would probably be better to write `December 2021` instead or `January 2022` here, on both line 224 and 225.

Reply:
A comment of the section below. Neither Java 1.7 nor Windows XP is relevant for JMRI anymore. Maybe remove the entire section? Windows XP is old.
Maybe even remove Java 8 from the lists below since it's no longer supported?

Reply:
Another comment on the table `OS and platform` which lists the last Java version that's supported on each platform.
- It might be good to update it, to `Doesn't support Java 11`, `Support Java 11` and `Support Java 17`.
- For me, the colors are confusing, but maybe they are selected for people that are color blind. But for me, darker red means "NO", light red means "Maybe" or "More or less". So when I see dark red means "able" and light red means "not able", there is a conflict in my brain. But I can live with it.

But this can be done later if desired.